test/tracerEq/test_v-diffusion_mes.py

```diff
@@ -177,36 +167,35 @@ def check_convergence(x_log, y_log, expected_slope, field_str, saveplot):
             plt.savefig(imgfile, dpi=200, bbox_inches='tight')
         if expected_slope is not None:
             err_msg = '{:}: Wrong convergence rate {:.4f}, expected {:.4f}'.format(setup_name, slope, expected_slope)
-            assert slope > expected_slope*(1 - slope_rtol), err_msg
+            assert slope > expected_slope, err_msg
             print_output('{:}: convergence rate {:.4f} PASSED'.format(setup_name, slope))
         else:
             print_output('{:}: {:} convergence rate {:.4f}'.format(setup_name, field_str, slope))
         return slope
 
-    check_convergence(x_log, y_log, polynomial_degree+1, 'salt', saveplot)
+    if expected_rate is None:
+        expected_rate = polynomial_degree+1
+    check_convergence(x_log, y_log, expected_rate, 'salt', saveplot)
 
 # ---------------------------
 # standard tests for pytest
 # ---------------------------
 
 
-@pytest.fixture(params=[0, 1])
-def polynomial_degree(request):
-    return request.param
-
-
-@pytest.fixture(params=[True, False], ids=['implicit', 'explicit'])
-def implicit(request):
-    return request.param
-
-
-@pytest.mark.parametrize(('stepper', 'use_ale'),
-                         [('LeapFrog', True),
-                          ('SSPRK22', True)])
-def test_vertical_diffusion(polynomial_degree, implicit, stepper, use_ale):
-    run_convergence([1, 2, 4], polynomial_degree=polynomial_degree, implicit=implicit,
-                    timestepper_type=stepper,
-                    use_ale_moving_mesh=use_ale)
+@pytest.mark.parametrize(
+    ('stepper', 'polynomial_degree', 'implicit', 'expected_rate'),
+    [
+        ('SSPRK22', 0, False, 1.0),
+        ('SSPRK22', 1, False, 1.7),
+        ('SSPRK22', 0, True, 1.0),
+        ('SSPRK22', 1, True, 2.2),
+    ]
+)
```

Comment:
Out of interest, how come you aren't testing leap frog any more?

Reply:
Primarily to cut down the testing. Also because leap-frog time integrator is not really being used and I think we can remove it soon.